src/test/java/com/code_intelligence/jazzer/mutation/mutator/StressTest.java

```diff
@@ -70,7 +70,7 @@
 public class StressTest {
   private static final int NUM_INITS = 1000;
   private static final int NUM_MUTATE_PER_INIT = 100;
-  private static final double MANY_DISTINCT_ELEMENTS_RATIO = 0.5;
+  private static final double MANY_DISTINCT_ELEMENTS_RATIO = 0.45;
```

Comment:
You can keep this as is and selectively use `distinctElementsRatio` with a lower bound on the individual tests that need it.